CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ execute_process(
     "${CMAKE_Swift_COMPILER}"
     -Xfrontend -disable-implicit-string-processing-module-import
     -Xfrontend -parse-stdlib
-    -c - -o /dev/null
+    -c - -typecheck
```

Comment:
Is the `-c` needed at all with `-typecheck`?  I don't think that we want to compile anything, we just can type-check the input.